### PR TITLE
Mhp 1630 my profile org id handling

### PR DESCRIPTION
**Pull request by jesusfreak3**

### CODE REVIEW

File: app/assets/javascripts/angular/missionhubApp.routes.js [hunk 1]
```diff
@@ -197,12 +197,12 @@ angular
                         },
                     ),
 
-                    organizationId: lazyLoadedResolve(
-                        /* @ngInject */ function($uiRouter) {
-                            var $transition$ = $uiRouter.globals.transition;
-                            return $transition$.params().orgId;
-                        },
-                    ),
+                    organizationId: $uiRouter => {
```
Comment (OzzieOrca) [inline app/assets/javascripts/angular/missionhubApp.routes.js]: I think you can inject `$transition$` directly. Maybe I'm wrong...

Comment (OzzieOrca) [reply app/assets/javascripts/angular/missionhubApp.routes.js]: I can't seem to open  the modal with the netlify link. Maybe you still need the `/* @ngInject */`. It tells angular-annotate to wrap it in that array syntax so when the minifier renames the variables it doesn't break dependency injection.

Comment (jesusfreak3) [reply app/assets/javascripts/angular/missionhubApp.routes.js]: > I think you can inject `$transition$` directly. Maybe I'm wrong...

I tried that, but when ever i tried it, the modal would never open, could not console to log either.  Wasnt sure why.  very weird behavoir

Comment (jesusfreak3) [reply app/assets/javascripts/angular/missionhubApp.routes.js]: Ok pushed with nginject will see if it fixes it.

**Pull request merged**